src/main/java/net/codejava/MvcController.java

```diff
@@ -311,6 +319,18 @@ public String submitWithdraw(@ModelAttribute("user") User user) {
                                                     userWithdrawAmtInPennies);
       jdbcTemplate.update(transactionHistorySql);
 
+      // calculate points 
+      String getUserCurrOverdraftToRewardsSql = String.format("SELECT CurrOverdraftToRewards FROM customers WHERE CustomerID='%s';", userID);
+      int userCurrOverdraftToRewardsSql = jdbcTemplate.queryForObject(getUserCurrOverdraftToRewardsSql, Integer.class);
+      int rewardsToAdd = (int)((newOverdraftAmtInPennies + userCurrOverdraftToRewardsSql)/(250*100));

```

Comment:
Make a constant for 250 and use the constant name instead

Reply:
Done